packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -413,6 +413,36 @@ export default class SalesforceCRMService implements CRM {
 
       let records: ContactRecord[] = [];
 
+      // This combination is for searching for ownership via contacts
+      if (
+        recordToSearch === SalesforceRecordEnum.CONTACT &&
+        appOptions?.roundRobinSkipFallbackToLeadOwner &&
+        forRoundRobinSkip
+      ) {
+        const searchResult = await conn.search(
```

Comment:
Throughout the codebase we use `conn.query` method but using `conn.search` allows you to return multiple record types.